src/status_im2/common/scroll_page/view.cljs

```diff
@@ -146,7 +146,7 @@
           [rn/view {:style {:height 151}}
            [rn/image
             {:source cover-image
-             :style  {:overflow :visible
+             :style  {:margin-bottom -16
```

Comment:
margin-bottom can't be negative, it doesn't look right

Reply:
In `scroll-view` child elements, the `overflow: visible` property will not work as expected on Android. However, using negative values for the margin property can provide a solution that works across both platforms, even though it is not ideal.

Reply:
Our goal is to overlap the bottom white area over the cover image. To achieve this instead of using a negative value for the `margin-bottom` property on this element, alternative solutions could be to utilize a negative value for either the `margin-top` or `top` property of the white area.

Reply:
@flexsurfer, I had the same problem recently and had to resort to the same workaround @ajayesivan did. I don't think there's another way, I've tried all sorts of things, but in the end, in our current RN version this is a known bug. https://github.com/facebook/react-native/issues/31218

@ajayesivan, have you tried passing the prop [removeClippedSubviews](https://reactnative.dev/docs/flatlist#removeclippedsubviews)? There's a big warning in the documentation, so even if that works we should carefully double-check for side-effects on different Android models.

Reply:
I tried `removeClippedSubviews`, but it didn't produce the desired result.